src/glossary.md

```diff
@@ -194,50 +194,50 @@ Exonum использует деревья Меркла и аналогичну
-для извлечения данных из [состояния блокчейна](#состояние-блокчейна). Обычно
-данные возвращаются с [доказательством](#доказательство-меркла) того, что данные
+[Конечная точка сервиса](#_17) которая может использоваться
+для извлечения данных из [состояния блокчейна](#_30). Обычно
+данные возвращаются с [доказательством](#_14) того, что данные
 действительно являются частью состояния блокчейна и авторизованы большинством
-[валидаторов](#валидатор).
+[валидаторов](#_7).
 
 ## Конечная точка сервиса
 
-Точка связи с [сервисом](#сервис). Существует три вида конечных точек:
+Точка связи с [сервисом](#_27). Существует три вида конечных точек:
 
-- [Транзакции](#транзакция) позволяют атомарно изменять состояние блокчейна;
-- [Запросы на чтение](#запрос-на-чтение) позволяют считывать данные из состояния
-  блокчейна, обычно вместе с [доказательством](#доказательство-меркла);
-- [Приватные API](#приватный-api) позволяют настраивать сервис локально.
+- [Транзакции](#_32) позволяют атомарно изменять состояние блокчейна;
+- [Запросы на чтение](#_16) позволяют считывать данные из состояния
+  блокчейна, обычно вместе с [доказательством](#_14);
+- [Приватные API](#api) позволяют настраивать сервис локально.
 
-Внешние объекты, такие как [легкие клиенты](#легкий-клиент) могут обращаться к
+Внешние объекты, такие как [легкие клиенты](#_21) могут обращаться к
 конечным точкам через REST API. Конфигурация REST API указана в сервисе.
 
 ## Консенсус
 
 **Также** алгоритм консенсуса
 
-Механизм достижения согласия между узлами в сети. В контексте [блокчейна](#блокчейн)
+Механизм достижения согласия между узлами в сети. В контексте [блокчейна](#_6)
 консенсусу необходимо противостоять ошибкам: ситуациям, когда узлы не реагируют
 и/или прямо пытаются скомпрометировать работу блокчейна.
 
 Существует два основных типа консенсуса:
 
-- [Аутентифицированный консенсус](#аутентифицированный-консенсус), в котором
+- [Аутентифицированный консенсус](#_4), в котором
   участвующие узлы известны заранее. Этот тип консенсуса реализован в Exonum;
 - Анонимный консенсус, в котором участники консенсуса заранее не известны. Этот
   тип консенсуса обычно используется в криптовалютах (например, Биткоин).
 
 ## Консенсусное сообщение
 
-[Сообщение](#сообщений), которое генерируется [полным узлом](#полный-узел)
-согласно [алгоритму консенсуса](#консенсус). Валидаторы обмениваются
-консенсусными сообщениями для принятия [транзакций](#транзакция) в блокчейн
-путем создания новых [блоков](#блок).
+[Сообщение](#_29), которое генерируется [полным узлом](#_24)
+согласно [алгоритму консенсуса](#_18). Валидаторы обмениваются
+консенсусными сообщениями для принятия [транзакций](#_32) в блокчейн
+путем создания новых [блоков](#_5).
 
 Консенсусные сообщения включают:
 
 - `Propose`, `Prevote` и `Precommit` сообщения, которые соответствуют 3
-  фазам [алгоритма консенсуса](#аутентифицированный-консенсус), используемого в
+  фазам [алгоритма консенсуса](#_4), используемого в
```

Comment:
I guess, here there should be link number 18

Reply:
Fixed